Tests/FBLPromisesTests/FBLPromise+DelayTests.m

```diff
@@ -37,19 +37,28 @@ - (void)testPromiseDelaySuccess {
     XCTAssertEqualObjects(value, @42);
     return value;
   }];
+  XCTestExpectation *delayedRejectCleanup = [self expectationWithDescription:@"delayedRejectCleanup"];
   FBLDelay(1, ^{
     [promise reject:[NSError errorWithDomain:FBLPromiseErrorDomain code:42 userInfo:nil]];
+    [delayedRejectCleanup fulfill];
   });
 
   // Assert.
   XCTAssert(FBLWaitForPromisesWithTimeout(10));
   XCTAssertEqualObjects(promise.value, @42);
   XCTAssertNil(promise.error);
+  
+  // Cleanup.
+  [self waitForExpectationsWithTimeout:1 handler:nil];
 }
 
 - (void)testPromiseDelayFail {
   // Act.
   FBLPromise<NSNumber *> *promise = [[FBLPromise resolvedWith:@42] delay:1];
+  XCTestExpectation *delayedResolveCleanup = [self expectationWithDescription:@"delayedResolveCleanup"];
```

Comment:
Nit: may like to use an empty string as expectation description everywhere, since there's only one expectation in every test case and when it fails the logged error is expressive enough. That can also help with keeping this line under 100 chars to fit it on one line.

Reply:
Done.